src/views/Gallery.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+  <div class="gallery-container">
+    <div class="gallery autoplay items-3">
+      <div id="item-1" class="control-operator"></div>
+      <div id="item-2" class="control-operator"></div>
+      <div id="item-3" class="control-operator"></div>
+
+      <figure class="item">
```

Comment:
Two things here: 
1. Let's make the carousel the full width of the page. 
2. The aspect ratio of the images should never be altered, because then they appear stretched/shrunk. This happens when I slide my dev tools across the screen.